.github/workflows/test-ansible.yml

```
@@ -7,12 +7,12 @@ jobs:
     name: e2e
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/setup-go@v3
-        with:
-          go-version: 1.19
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - uses: actions/setup-go@v4
+        with:
+        go-version-file: "go.mod"
```

Is indentation off here?
```suggestion
          go-version-file: "go.mod"
```